runhouse/servers/cluster_servlet.py

```diff
@@ -330,6 +336,37 @@ async def acheck_cluster_logs(self, interval_size: int):
 
         return logs_den_resp, new_start_log_line, new_end_log_line
 
+    async def aperiodic_autostop_check(self):
+        """Periodically check the autostop of the cluster"""
+        while True:
+            should_update_autostop: bool = self.autostop_helper is not None
+            cluster_config = await self.aget_cluster_config()
+
+            # Note: for now using the same interval for both autostop and status checks
+            interval_size = cluster_config.get(
+                "status_check_interval", DEFAULT_STATUS_CHECK_INTERVAL
+            )
+
+            if not should_update_autostop:
+                logger.debug(
+                    f"Not updating autostop, checking again in {interval_size} minutes"
```

Comment:
I don't think this debug statement makes sense since `should_update_autostop` as determined currently shouldn't ever change across loops. it's just based on whether or not the cluster is an ondemand cluster.

also noting that `break` here breaks out of the while loop. should be `continue` if we intend to continue running the thread in the background